packages/datadog-plugin-aws-sdk/test/dynamodb.spec.js

```diff
@@ -382,37 +375,38 @@ describe('Plugin', () => {
                         Value: { S: 'bar4' }
                       }
                     }
-                  }, callback)
+                  })
                 }
               })
             })
 
-            it('should add span pointer for deleteItem', () => {
-              testSpanPointers({
+            it('should add span pointer for deleteItem', async function () {
+              return testSpanPointers({
                 expectedHashes: '27f424c8202ab35efbf8b0b444b1928f',
-                operation: (callback) => {
-                  dynamo.deleteItem({
+                operation () {
+                  return util.promisify(dynamo.deleteItem).bind(dynamo)({
```

Comment:
Could you please use a helper method that checks if the method already supports promises and to use that instead in those cases? That way we prevent the deprecation warning from popping up and having to fix this later.